mesh/mesh.hpp

```diff
@@ -1330,7 +1330,9 @@ class Mesh
        element @a elem, including @a elem. */
    Array<int> FindFaceNeighbors(const int elem) const;
 
-   /// Return the index and the orientation of the face of bdr element i. (3D)
+   /** Return the index and the orientation of the vertex of bdr element i. (1D)
+       Return the index and the orientation of the edge of bdr element i. (2D)
+       Return the index and the orientation of the face of bdr element i. (3D) */
    void GetBdrElementFace(int i, int *f, int *o) const;
```

Comment:
We should probably note that in 2D this method returns orientations 0/1, and not +1/-1 (as does `GetBdrElementEdges`).

Reply:
[2d83759](https://github.com/mfem/mfem/pull/3943/commits/2d83759e49f74d261ffeb41091f6db24111fe34c)